File: MPI/mpicheck/tests/functions/2.0/MPI_Lookup_name.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    const char *var_0;
    MPI_Info var_1;
    char *var_2;
    int ret;
    /* calls */
    ret = MPI_Lookup_name(var_0, var_1, var_2);
    ret = PMPI_Lookup_name(var_0, var_1, var_2);
    return 0;
}

File: MPI/mpicheck/tests/functions/2.0/MPI_Lookup_name.f90

        
        program main
        use mpi
        
        CHARACTER*(10) var_0
       INTEGER var_1
       CHARACTER*(10) var_2
       INTEGER var_3
        call mpi_lookup_name(var_0, var_1, var_2, var_3)
       call pmpi_lookup_name(var_0, var_1, var_2, var_3)
        end program main
    